packages/ui/src/AppHeader/EnvironmentBadge/EnvironmentBadge.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Box } from 'react-feather'
+
+import { Icon } from '../../Icon'
+
+import styles from './EnvironmentBadge.module.scss'
+
+export interface EnvironmentBadgeProps {
+  environment: string
```

Comment:
We had a plan for assigning different colors to different environments. Taking `{ environment: string, color: string }` might be useful to pass it as style.

Reply:
Currently this is just a placeholder.